server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -212,8 +223,84 @@
 
         return row;
       });
+      if (reportType.bespokeReport && reportType.bespokeReport.insertRows) {
+        Object.keys(reportType.bespokeReport.insertRows).map((key) => {
+          if (key === 'last') {
+            rows.push(reportType.bespokeReport.insertRows[key](tableData))
+          } else {
+            rows.splice(key, 0, reportType.bespokeReport.insertRows[key](tableData));
+          }
+        });
+      }
+      return rows;
     };
 
+    const buildStandardTable = function(reportType, tableData, tableHeadings, sectionHeading = '') {
+      let tableRows = [];
+      const tableHeaders = buildTableHeaders(reportType, tableHeadings);
+
+      if (reportType.grouped) {
+        let longestLength = 0;
+        for (const [header, data] of Object.entries(tableData)) {
+          let group = buildStandardTableRows(data, tableHeadings);
+          let link;
+
+          if (reportType.grouped.headings && reportType.grouped.headings.link) {
+            if (reportType.grouped.headings.link === 'pool-overview') {
+              link = app.namedRoutes.build('pool-overview.get', {poolNumber: header});
+            }
+          }
+
+          longestLength = group[0].length > longestLength ? group[0].length : longestLength; 
```

Comment:
would it make more sense to rename this as `longestGroup` or `longestGroupItem`, since that is what we are checking for?

Reply:
updated